Tests/RxSwift/fromAsyncTests.swift

```diff
@@ -36,4 +44,47 @@ class FromAsyncTests: XCTestCase {
 
         XCTAssertEqual(observer.events, correct)
     }
+    
+    func testSingleResultEqualitySuccessCase() {
+        // given
+        let result = "result"
+        let expectedEvents = [next(0, result), completed(0)]
```

Comment:
Please don't use the global deprecated `next` and `completed` functions

Reply:
I can't see any warning about deprecated methods usage. I also can't find deprecation in docs. Could u please attach some proof, that this functions were deprecated?

Reply:
It's a bit implicit, but here: [RxTest/Deprecated.swift](https://github.com/ReactiveX/RxSwift/blob/master/RxTest/Deprecated.swift#L35)

Reply:
The deprecations are runtime deprecations but they've been there for a while.

Reply:
Thanks, resolved